src/views/OfflineDatalog.vue

```diff
@@ -1,21 +1,39 @@
 <template>
   <div class="Graph">
-    <Dropdown
-      class="channel-dropdown"
-      :options="channelsList"
-      :selected="selectedChannel"
-      :placeholder="'Select channel to plot'"
-      v-on:updateOption="onSelectedChannel"
-    >
-    </Dropdown>
-    <div class="chart-container">
-      <datalog-chart ref="datalogChart" />
-    </div>
-    <div id="container">
-      {{ offlineDatalogStatus }}
-      {{ computePacket }}
+    <ul class="bt-container">
+      <div class="sync-container">
+        <button class="sync-bt" @click="syncOfflineDatalogRecords()">
+          Sync Data
+        </button>
+      </div>
+      <div class="delete-container">
+        <button class="delete-bt" @click="$vm2.open('modal')">
+          Delete data from GoGoBoard
+        </button>
+      </div>
+    </ul>
+    <div class="datapicker">
+      <div class="datepciker-container">
+        <date-picker
+          v-model="dateTimeOffset"
+          type="datetime"
+          placeholder="Select datetime "
+          value-type="timestamp"
+          v-on:updateOption="timestamp"
```

Comment:
does this works ? may remove before merged

Reply:
Yes , you can remove `v-on:updateOption`. it not work in datepicker